tests/acceptance/fixtures.py

```diff
@@ -579,21 +579,21 @@ def test_foo():
 
 
 @pytest.fixture(autouse=True)
-def only_with_distro_feature(request, bitbake_variables):
-    """Fixture that enables use of `only_with_distro_feature(feature1, feature2)` mark.
+def only_with_mender_feature(request, bitbake_variables):
+    """Fixture that enables use of `only_with_mender_feature(feature1, feature2)` mark.
```

Comment:
You probably know, but some tests need an update accordingly.

Reply:
Lol, I went out of my way to make sure all the `DISTRO_FEATURES` variables were changed, and completely forgot this part!

Reply:
Hehe. I first thought that it was so obvious that I shouldn't comment on it... :smile: